protobuf/api/event.proto

```diff
@@ -45,7 +46,7 @@ message CreateEventRequest {
   string key = 2;
 
   // data is the data for the event.
-  string data = 3;
+  google.protobuf.Struct data = 3;
```

Comment:
As the type changed, we should also change the number:
```suggestion
  google.protobuf.Struct data = 4;
```
Need to be done everywhere else

Reply:
anyway we don't support the string so the previous api breaks anyway so let's not add backward compatibility on the api when we don't support it in the software yet

Reply:
+1 for leaving api id as it is.